app/templates/fd-editform-constructor.hbs

```diff
@@ -27,4 +26,78 @@
   {{#each model.controls as |control|}}
     {{fd-editform-row row=control selectAction=(action "selectControl")}}
   {{/each}}
-</form>
\ No newline at end of file
+</form>
+
+{{#ui-sidebar
```

Comment:
To avoid him moving to the body and error from Semantic UI, he needs to specify the right context through `ui_context` property.